Source/Dafny/ConcreteSyntax/ConcreteSyntaxTree.cs

```diff
@@ -7,10 +7,11 @@
 
 namespace Microsoft.Dafny {
   public enum BraceStyle {
```

Comment:
Since you're already touching most of the references to this type, I think the time is right to fix the name and call it `BlockStyle` instead. That will resolve the weird redundancy in `BraceStyle.Brace` and just plain contradiction in `BraceStyle.NewlineNoBrace` (which would just become `BlockStyle.Newline`).

I think then `Space` would also be renamed to `SpaceBrace`?